src/models/animal.model.ts

```diff
@@ -0,0 +1,23 @@
+import { sequelize } from '../db/connection'
+import { Model, InferAttributes, InferCreationAttributes, CreationOptional, DataTypes } from 'sequelize'
+
+export class Animal extends Model<InferAttributes<Animal>, InferCreationAttributes <Animal>> {
+  declare id: CreationOptional<number>
+  declare name: string
+}
+Animal.init({
+  id: {
+    type: DataTypes.UUID,
+    autoIncrement: true
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false
+  }
+}, {
+  sequelize,
+  modelName: 'animal'
```

Comment:
i think this line is optional
sequalize creates the table with the model name in plural
so it's not preferred to override the names in sequalize